infoschema/tables.go

```diff
@@ -637,6 +636,10 @@ func dataForTables(ctx sessionctx.Context, schemas []*model.DBInfo) [][]types.Da
 	createTimeTp := tablesCols[15].tp
 	for _, schema := range schemas {
 		for _, table := range schema.Tables {
+			collation := table.Collate
+			if collation == "" {
+				collation = charset.CollationUTF8
```

Comment:
Check the value of charset, the decide whether use CollationUTF8MB4 or CollationUTF8.

Reply:
Could we replace it with `mysql.DefaultCollationName`?

Reply:
Yeah, but they are exactly same. - - 7